tests/reporter/kubernetes.go

```diff
@@ -302,6 +303,56 @@ func (r *KubernetesReporter) logAuditLogs(virtCli kubecli.KubevirtClient, since
 	}
 }
 
+func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, duration time.Duration) {
+
+	const component = "journal"
+
+	logsdir := filepath.Join(r.artifactsDir, "nodes")
+	if err := os.MkdirAll(logsdir, 0777); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create directory %s: %v\n", logsdir, err)
+		return
+	}
+
+	logDuration := strconv.FormatInt(int64(duration/time.Second), 10)
+
+	nodes := getNodesWithVirtLauncher(virtCli)
+
+	for _, node := range nodes {
+		pod, err := kubecli.NewVirtHandlerClient(virtCli).Namespace(tests.KubeVirtInstallNamespace).ForNode(node).Pod()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to get virt-handler pod on node %s: %v", node, err)
+			return
```

Comment:
continue, same for the others, so node error wont stop them all

Reply:
Done